### ISSUE TITLE
Sprint 76 - Public Websites

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#12194]
### 1. Harden homepage for prod

**Page itself:**
  - [ ] (stretch) #12090 ()
  * **Analytics:**
    - [ ] Tracking Analytics Team dependency in #11965 (0)
    - [ ] GA fix for existing homepage -- Analytics team GTM work required (0)
    - [ ] GA fix for new homepage -- Analytics team GTM work required (0)

**Drupal:**
  - [ ] (stretch; pending CMS Collab Cycle) #11692 ()

### 2. Header/footer redesign
- Awaiting green light

### 3. Income Limits web app
- [x] #12158 (8)
- [x] #12163 (8)
- [x] #12219
- [x] #12218
- [x] #12249 

### 4. Benefit Content Model Hardening
- [ ] (DaveP) #12058 ()

### 5. PACT Act Wizard
- Chatbot leading content

### 6. Accordion header levels
- [ ] #12183 🥅 
- [ ] #12255 (8)

### 7. Cerner API deprecation support
- [ ] (blocked in progress s76) #9078 
- [ ] (blocked on dependent team) #9307 

###  Other 508
- [ ] #12169 

---

### Redirects (3 total)
- [x] #12245 
- [ ] #12246 
- [ ] #12247 

### Tech debt
- [ ] #10735 ()
- [ ] (stretch) #10307 
- [ ] #12220 (3)
  - possible follow-on(s) to 12220
- [ ] (stretch; from Samara) #12195 

### Drupalize footer - 8 FE points
- [ ] (stretch) department-of-veterans-affairs/va.gov-cms#11482 (8)

### Injected header post-mortem
- Actions waiting on conclusion of post-mortem

## TOTALS: 
FE: 
Drupal: 
Design:

**Comment by jilladams** [comment]
Redirect, May come to us for S76: https://github.com/department-of-veterans-affairs/va.gov-team/issues/51597

**Comment by jilladams** [comment]
Income limits: 
* Q1 plan - target for this sprint = get to the point of a complete discussion guide ready for review by Conlon / mid-point reviewers and ready for Shane review from there for recruiting purposes.
  * AR: Update https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12248 with narrative / ACs.
* Architecture: WIP. Could begin API design work, but want to first hear from Platform if our plan is doable. 

Dave notes: 
* CMS Collab cycle re: hardened content model for kickoff 

Accordion headers: 
* Entity ID  /hard-coded solve is not large enough for remediating all 17k. Can be used now as band-aid for burials issue
* Dave is interested in a short term approach + long-term. Hard code until full scope
* **AR:** Need a new ticket for 5-8 points to cover planning for the real fix

Redirects: 
* https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12245
* https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12246
* https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12247

**Comment by chri5tia** [comment]
Some work I can take on:
Sprint 76 ideas

Check and define next steps:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4692
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9469
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4708

Ready to work on:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10238
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9611

**Comment by jilladams** [comment]
Stretch: #12288

**Comment by jilladams** [comment]
#12292 another stretch option
